### ISSUE TITLE
Weird Title Bug

### ISSUE THREAD

**Issue by bendangelo** [nvbn/everpad#108]
To reproduce create a new note. In the title text field press escape. You will see a weird box with numbers appear in the title bar.

**Comment by nvbn** [comment]
What distro and version are you using?
What version of pyside?
I can't reproduce it in fresh ubuntu.

**Comment by bendangelo** [comment]
Ubuntu 12.04. How do I get the version of pyside?

Forgot to mention the issue will appear in the title bar. Pressing the escape key in the title text field will enter in invisible characters but they are visible in the title bar / note title afterwards. I can take a screenshot too.

**Comment by fmoo** [comment]
I can sort of repro this.  I'm on PySide 1.1.0

While I can't see the weird box with numbers in the title anymore (I used to be able to see them, but it appears this was fixed in df50e637e558f5698f1f7cfedf5b010bc5c7c959), I can still tell they are there by using the keyboard left/right to navigate around  (instead of going to the previous char, the nul bytes are inserted into the content area)

Basically, it looks like contenteditable=true is not eating the <esc> presses and is instead inserting some nul / control bytes

**Comment by hdave** [comment]
I think I've seen this too.  Is this the issue? Check out the title bar of the note:

https://dl.dropbox.com/u/3891104/Screenshot%20from%202012-10-24%2010%3A28%3A13.png

**Comment by bendangelo** [comment]
Looks like escaped html. I'm not seeing that. I see a box with numbers in it when pressing escape. Try hitting escape.

**Comment by fmoo** [comment]
@hdave that issue should be fixed in fd0bcdbd560d2ffa91757aa57a55dfa5f200b387

df50e637e558f5698f1f7cfedf5b010bc5c7c959 fixes the junk from being displayed in the title bar, but it's still getting injected into the title/body (albeit invisibly).

We should drop those keystrokes on the floor (`return false`) in the keypress handler i think.  Shouldn't be too hard a change.

**Comment by bendangelo** [comment]
Just updated to the latest version and this has been fixed.

**Comment by fmoo** [comment]
Actually, I don't think this is 100% fixed yet.  Pressing [esc] still inserts characters into the document, it's just that they are no longer rendered
